src/irc.lua

```diff
@@ -41,12 +56,38 @@ irc.modeset = function (target, recipient, mode)
     end
 end
 
-irc.kick = function (target, recipient, message)
-    if target:byte() == 35 then
-        irc.connection:send(('KICK %s %s :%s\r\n'):format(target, recipient, message))
-    else
-        irc.privmsg(target, 'Cannot kick in query')
-    end
+irc.names = function (channel)
+    irc.connection:send(('NAMES %s\r\n'):format(channel))
+end
+
+irc.nickset = function (nickname)
```

Comment:
I think, generally, it is better to have the function-name reflect the exact verb where possible. As a result, I would suggest ``nick()`` over ``nickset()``.

Reply:
I used set because you have irc.modeset :P I was trying to follow some kind of convention based off the other functions XD

If you want nickset changed to nick, should modeset also be changed...?

Reply:
Yes please!

Reply:
done :dango: